x/gov/types/params.go

```diff
@@ -74,11 +74,12 @@ func ParamKeyTable() paramtypes.KeyTable {
 }
 
 // NewDepositParams creates a new DepositParams object
-func NewDepositParams(minDeposit sdk.Coins, maxDepositPeriod time.Duration, minExpeditedDeposit sdk.Coins) DepositParams {
+func NewDepositParams(minDeposit sdk.Coins, maxDepositPeriod time.Duration, minExpeditedDeposit sdk.Coins, minInitialDepositPercent uint32) DepositParams {
```

Comment:
Let's avoid changing function signatures on functions in the cosmos-sdk repo if possible.

To avoid situations like this: https://github.com/osmosis-labs/cosmos-sdk/issues/279